05_syntax_lab/02.py

```diff
@@ -1,6 +1,8 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+from random import randint
+sum = 0
+for i in range(7):
+    x = randint(1,100)
+    sum += x

```

Comment:
Notice how "sum" is also a function name (like "max"), so sum is coloured differently than x